src/containers/shared/components/Transaction/Clawback/TableDetail.tsx

```diff
@@ -0,0 +1,25 @@
+import { useTranslation } from 'react-i18next'
+import { Amount } from '../../Amount'
+import { TransactionTableDetailProps } from '../types'
+import { ClawbackInstructions } from './types'
+import { Account } from '../../Account'
+
+export const TableDetail = ({
+  instructions,
+}: TransactionTableDetailProps<ClawbackInstructions>) => {
+  const { t } = useTranslation()
+  const { amount, holder } = instructions
+
+  return (
+    <div>
+      {amount && holder && (
+        <div className="clawback">
+          <span className="label">{t('claws_back')}</span>
+          <Amount value={amount} displayIssuer />
+          <span>{t('from')}</span>
+          <Account account={holder} />
+        </div>
```

Comment:
IMO this should all be one translation

Reply:
i dont think so. The `claws_back` translation needs the `className="label"` in order to appear in bold

Reply:
You can use the `<Trans>` component to accomplish this as one translation.

Reply:
fixed